reference/api/pubsub_api.md

```diff
@@ -85,4 +90,4 @@ A JSON encoded payload.
 
 ## Message Envelope
 
-Dapr Pub/Sub adheres to version 0.3 of Cloud Events.
+Dapr Pub/Sub adheres to version 1.0 of Cloud Events.
```

Comment:
While we're updating this howto, can you please add a "Related Links" section as per this: https://github.com/orizohar/docs/blob/docs-guidance/contributing/howto-template.md

Reply:
Done